nomenclature/definitions/subannual/README.md

```diff
@@ -43,6 +43,59 @@ with open(f'../subannual/months.yaml', 'r') as stream:
 mapping = dict([(m, eval(attr['duration'])) for (m, attr) in months.items()])
 ```
 
+### Daily and hourly data
+Our convention makes reference to the following agreements:
+
+
+#### UTC datetime format:
+- Standard datetime format     : **UTC**  
+> The UTC datetime format is adopted to consider the following levels: year, month, day, hour, minute, and second (without consider time zones).
+> For example: `2020-01-01, 2020-01-02, ...` or `2020-01-01T13:00, ...`
+> The use of lists of dates and datetimes will be generated by the codes of the next section.
+- Using time zone                    : **no relevance**
```

Comment:
I think writing "no relevance" is misleading - this might be clearly relevant for some contexts, but we should rather explain that we do not include special treatment for identifying time zones and assume everything is in UTC-Z.

Reply:
OK, I got it.